openhands/runtime/README.md

```diff
@@ -89,9 +89,27 @@ Key features:
 - Real-time logging and debugging capabilities
 - Direct access to the local file system
 - Faster execution due to local resources
+- Container isolation for security
 
 This is the default runtime used within OpenHands.
 
+### Local Runtime
+
+The Local Runtime is designed for direct execution on the local machine:
+
+- Runs the action_execution_server directly on the host
+- No Docker container overhead
+- Direct access to local system resources
+- Ideal for development and testing when Docker is not available or desired
+
+Key features:
+- Minimal setup required
+- Direct access to local resources
+- No container overhead
+- Fastest execution speed
+
+Note: This runtime provides less isolation than the Local Docker Runtime.
```

Comment:
@openhands-agent Does the new Local Runtime provide less isolation than the Local Docker Runtime, or no isolation at all, basically? If your answer is no isolation, then the documentation should say that clearly.